clusterloader2/cmd/clusterloader.go

```diff
@@ -50,6 +48,8 @@ import (
 const (
 	dashLine        = "--------------------------------------------------------------------------------"
 	nodesPerClients = 100
+	NodeExternalIP  = "ExternalIP"
+	NodeInternalIP  = "InternalIP"
```

Comment:
I don't like this change - let's use the constants from corev1 instead of introducing our own ones.

Reply:
This is the error we are getting when used the corev1 constants...
 
perf-tests/clusterloader2/cmd/clusterloader.go:153:54: cannot use corev1.NodeExternalIP (constant "ExternalIP" of type v1.NodeAddressType) as v1.NodeAddressType value in argument to util.GetMasterIPs

Reply:
This is strange - but probably some casting would be enough to fix. Let's not introduce new constants for existing ones.